### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by jlovejoy**
Updated incorrect reference to Public Binary License (should be permissive) and updated header notice to better wording

### CODE REVIEW

File: docs/cont/contributing.md [hunk 1]
```diff
@@ -74,20 +74,20 @@ In order to clearly reflect the PBL license, please create three text files:
 <pre>Unless specifically indicated otherwise in a file, files are licensed under the Public Binary License,
```
Comment (adamgreen) [inline docs/cont/contributing.md]: Still says Public Binary License on this line.

Comment (AnotherButler) [inline docs/cont/contributing.md]: @jlovejoy Should we change line 74 to Permissive Binary License, too?

Comment (jlovejoy) [inline docs/cont/contributing.md]: yes, fixed, missed that one, obviously!

**Comment by AnotherButler** [comment]
Thanks so much!

**Pull request merged**